src/ruby.rs

```diff
@@ -947,6 +947,22 @@ extern {
     ///     [variable.c](https://github.com/ruby/ruby/blob/v2_5_1/variable.c#L459-L463)
     /// * **2.6:** [ruby.h](https://github.com/ruby/ruby/blob/v2_6_0_preview2/include/ruby/ruby.h#L1808)
     pub fn rb_obj_classname(obj: VALUE) -> *const c_char;
+
+    /// Equivalent of `Kernel#block_given?`.
+    /// Returns 1 (true) if yield would execute a block in the current context, or 0 (false)
+    /// otherwise.
```

Comment:
I also wonder if an explanation of what "context" means here would be helpful, though I guess the `Kernel#block_given?" docs don't explain either. I feel like I have a bit better notion of what context is in Ruby itself, but maybe a little less so when we're using the C interface.

Reply:
"current method" is probably more accurate. It is probably UB if you call this outside of a method (if you can even figure out a way to do that, maybe in your `Init_*`)